bridge-impl/src/main/java/com/liferay/faces/bridge/scope/internal/BridgeRequestScopeWrapper.java

```diff
@@ -38,11 +37,6 @@ public PortletPhase getBeganInPhase() {
 	}
 
 	@Override
-	public long getDateCreated() {
```

Comment:
@stiemannkj1 This is outstanding! Incorporating the LRU feature into the Cache via `CachedValue` makes it possible to get rid of this. Excellent -- shows that the design is elegant.